_sass/all.scss

```diff
@@ -76,6 +76,11 @@ a {
         font-size: small;
         color: #404040;
     }
+    
+    .username {
+        font-size: medium;
+        color: #404040;
```

Comment:
Changin hard-coded color to variables would be better

Reply:
hey, the changes made by you will be inherited in our pull requests,  once your pull request gets merged!